drivers/hmis/app/graphql/mutations/update_beds.rb

```diff
@@ -0,0 +1,23 @@
+module Mutations
+  class UpdateBeds < BaseMutation
+    argument :inventory_id, ID, required: true
+    argument :bed_ids, [ID], required: true
+    argument :name, String, required: false
+    argument :unit, ID, required: false
+
+    field :beds, [Types::HmisSchema::Bed], null: false
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def resolve(inventory_id:, bed_ids:, name: nil, unit: nil)
+      inventory = Hmis::Hud::Inventory.editable_by(current_user).find_by(id: inventory_id)
+      return { beds: [], errors: [InputValidationError.new('Inventory record not found', attribute: 'inventory_id')] } unless inventory.present?
+
+      beds = inventory.beds.where(id: bed_ids)
+      common = { user_id: hmis_user.user_id, updated_at: Time.now }
+      beds.update_all(name: name, **common) if name.present?
+      beds.update_all(unit_id: unit.to_i, **common) if unit.present?
```

Comment:
I think this means you can't remove a name or unit, does that match your understanding of the workflow?  (I _think_ that's good, just want to confirm)

Reply:
Yeah that's correct